net-doc/net/scribblings/dns.scrbl

```diff
@@ -32,6 +32,31 @@ an error will be raised.
 }
 
 
+@deftogether[(
+@defproc[(dns-get-srv [nameserver string?]
+		      [name string?]
+		      [service string?]
+		      [proto string? "tcp"])
+         (listof srv-rr?)]
+@defstruct*[srv-rr ([priority (integer-in 0 65535)]
+		    [weight (integer-in 0 65535)]
+		    [port (integer-in 0 65535)]
+		    [target string?]) #:prefab]
+)]{
+
+Consults the specified nameserver (normally a numerical address like
+@racket["128.42.1.30"]) to retrieve the
+@hyperlink["https://en.wikipedia.org/wiki/SRV_record"]{SRV records}
```

Comment:
Maybe the SRV ref should be a margin note? (ala math library) Mainly because there are rarely external hyperlinks in the docs and to not make it look like an internal link.